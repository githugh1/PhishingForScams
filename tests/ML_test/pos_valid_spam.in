{
  "email_subject": "You've Won a Prize",
  "email_body": "Congratulations! You've won a million dollars...",
  "sender": "lottery@email.com"
}
