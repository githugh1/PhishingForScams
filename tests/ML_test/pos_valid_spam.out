{
  "is_spam": true,
  "confidence": 0.9,
  "message": "This email is likely spam."
}
